scipy/sparse/matrix_io.py

```diff
@@ -0,0 +1,149 @@
+import numpy as np
+import scipy.sparse
+
+
+def save(file, matrix, compressed=True):
+    """ Save a sparse matrix.
+    
+    Save the sparse matrix into a file in ``.npz`` format.
+    If compressed, the resulting file is compressed.
+
+    Parameters
+    ----------
+    file : str or file
+        Either the file name (string) or an open file (file-like object)
+        where the data will be saved. If file is a string, the ``.npz``
+        extension will be appended to the file name if it is not already
+        there.
+    matrix: spmatrix (format: ``csc``, ``csr`` or ``bsr``)
+        The sparse matrix to save.
+    compressed : bool, optional
+        Allow compressing the file. Default: True
+
+    Returns
+    -------
+    None
+
+    See Also
+    --------
+    numpy.savez: Save several arrays into a ``.npz`` archive.
+    numpy.savez_compressed : Save several arrays into a compressed ``.npz`` archive.
+
+    Notes
+    -----
+    The ``.npz`` file format is a zipped archive of files named after the
+    variables they contain. The archive containts the three ``.np`` files
+    ``data``, ``indices`` and ``indptr`` containing the corresponding values
+    of the sparse matrix.
```

Comment:
This note is specific to certain kinds of sparse matrix formats. I think we can just remove the whole "Notes" section entirely.